apollo-router/tests/tracing_tests.rs

```diff
@@ -38,6 +38,7 @@ async fn make_request(request: supergraph::Request) {
 }
 
 #[test_span(tokio::test)]
+#[level(tracing::Level::ERROR)]
```

Comment:
same as in `test_harness.rs`: we need to change the filters due to salsa logs appearing in the traces